backend/curriculum_tracking/management/commands/fix_project_and_topic_complete_times.py

```diff
@@ -0,0 +1,28 @@
+#update the completed time on all projects and topics
+
+from django.core.management.base import BaseCommand
+from curriculum_tracking.models import RecruitProject,AgileCard,RecruitProjectReview
+from curriculum_tracking.constants import REVIEW_STATUS_CHOICES
+
+
+
+
+def fix_complete_times():
+    cards = AgileCard.objects.filter(status=AgileCard.COMPLETE)
+
+    for card in cards:
+        project = card.recruit_project
+        topic = card.topic_progress
+        if project:
+            project.complete_time = project.latest_review()
```

Comment:
complete_time is a datetime field, latest_review returns a review object.
This would work:
```
project.complete_time = project.latest_review().timestamp 
```